src/game.py

```diff
@@ -1,2 +1,18 @@
 def start():
     return 'Welcome to the 15 puzzle game.'
+
+SIZE = 4
```

Comment:
SIZE is a too generic name. Can you find a better constant name?

Moreover, move the constants on the top of the file